courses/free-courses-en.md

```diff
@@ -800,6 +801,16 @@
 * [Vuex for Everyone](https://vueschool.io/courses/vuex-for-everyone)
 
 
+### Jenkins
+
+* [FREE Advanced Jenkins in K8s (Docker in Docker)](https://www.udemy.com/course/free-advanced-jenkins-in-k8s-docker-in-docker) - Nomad Sashi (Udemy)
+* [Jenkins](https://youtube.com/playlist?list=PLhW3qG5bs-L_ZCOA4zNPSoGbnVQ-rp_dG) - Raghav Pal (YouTube)
+* [Jenkins Tutorial](https://www.mygreatlearning.com/academy/learn-for-free/courses/jenkins-tutorial) (Greate Learning)
+* [Jenkins Tutorial Step by Step](https://www.youtube.com/playlist?list=PL8VbCbavWfeGt_Hyq0h9Pymi9DTGowd5X) - Pramod Dutta (YouTube)
+* [Jenkins Tutorial Video [2022 updated]](https://www.youtube.com/playlist?list=PLEiEAq2VkUUKGrfcoNYRgqam5YBERN8xa) - Matthew (Youtube)
+* [Learning Jenkins (2018)](https://www.linkedin.com/learning/learning-jenkins-14423877?replacementOf=learning-jenkins-2018) (Linkedin)
```

Comment:
Looks like this course is not free

Reply:
* [Jenkins Tutorial](https://www.mygreatlearning.com/academy/learn-for-free/courses/jenkins-tutorial) (Greate Learning)
 --->I checked this course and it is free.
![ss1](https://user-images.githubusercontent.com/112818612/195641048-a52970e3-2348-450f-be97-8138bd4fff84.png)

* [Jenkins Tutorial Step by Step](https://www.youtube.com/playlist?list=PL8VbCbavWfeGt_Hyq0h9Pymi9DTGowd5X) - Pramod Dutta (YouTube)
  ---> As this one is from youtube and it is free.
![ss2](https://user-images.githubusercontent.com/112818612/195641730-3f4e3b78-ee33-4ccb-9b72-1a5288752111.png)

* [Jenkins Tutorial Video [2022 updated]](https://www.youtube.com/playlist?list=PLEiEAq2VkUUKGrfcoNYRgqam5YBERN8xa) - Matthew (Youtube)
  ---> As this one is also from youtube and it is free.
![ss3](https://user-images.githubusercontent.com/112818612/195642396-4a277497-07d7-4e81-abfe-13cea591bfa1.png)

Reply:
And the last course of linkedin so sorry about that it is only free for a month.
So i am going the replace it with another new course.